src/main/java/at/hannibal2/skyhanni/utils/RenderUtils.kt

```diff
@@ -33,7 +34,7 @@ import kotlin.time.Duration
 import kotlin.time.DurationUnit
 
 object RenderUtils {
-
+    enum class HorizontalAlignment{LEFT, CENTER, RIGHT}
     private val beaconBeam = ResourceLocation("textures/entity/beacon_beam.png")
```

Comment:
Only a simple formatting nitpick
```suggestion
    
    enum class HorizontalAlignment{LEFT, CENTER, RIGHT}
    
    private val beaconBeam = ResourceLocation("textures/entity/beacon_beam.png")
```